core/platform/bulk_email/mailchimp_bulk_email_services.py

```diff
@@ -86,7 +86,9 @@ def _create_user_in_mailchimp_db(
     Args:
         client: mailchimp3.MailChimp. A mailchimp instance with the API key and
             username initialized.
-        subscribed_mailchimp_data: dict. Post body with required fields.
+        subscribed_mailchimp_data: dict. Post body with required fields for a
+            new user. The required fields are email_address, status and tags
```

Comment:
Second sentence is run-on, not sure whether to read it as "The required fields are email_address, status and tags" or "tags with any relevant merge_fields as optional". Could you please clarify?

Reply:
Done